src/index.html

```diff
@@ -109,7 +113,7 @@
       </ol>
       <p class="total">
         Total:
-        <span class="total-population">Calculate it!</span>
+        <span class="total-population">${total}</span>
```

Comment:
The total population is displayed as `${total}`. Ensure that this placeholder is replaced with the actual total population value, possibly calculated using JavaScript.